static/src/js/containers/MapContainer/MapWrapper.js

```diff
@@ -93,65 +90,24 @@ const MapWrapper = ({
     }
   }, [])
 
-  const [gibsLayer, setGibsLayer] = useState('')
-  const [colorMap, setColorMap] = useState({})
-
-  // TODO EDSC-3880 We can move the below state relating to colormaps into Redux which might help when eventually
-  // managaging multiple GIBS layers.
-  useEffect(() => {
-    const { tags } = focusedCollectionMetadata
-    const gibsTag = getValueForTag('gibs', tags)
-
-    if (gibsTag) setGibsLayer(gibsTag[0])
-  }, [focusedCollectionMetadata])
+  const { tags } = focusedCollectionMetadata
+  const [gibsTag] = getValueForTag('gibs', tags) || []
 
   // Check that we are in the correct projection
-
   const hasGibsLayerForProjection = (gibsLayer, projection) => {
     if (projection === projections.arctic && gibsLayer.arctic) return true
     if (projection === projections.geographic && gibsLayer.geographic) return true
     if (projection === projections.antarctic && gibsLayer.antarctic) return true
     return false
   }
 
-  useEffect(() => {
-    const getColorMap = async () => {
-      const hasNoColormapForProjection = (
-        !colorMap[focusedCollectionId]
-        || (colorMap[focusedCollectionId] && !colorMap[focusedCollectionId][projection])
-      )
-
-      if (
-        gibsLayer
-        && gibsLayer.product
-        && hasGibsLayerForProjection(gibsLayer, projection)
-        && hasNoColormapForProjection
-      ) {
-        try {
-          const response = await fetch(`${apiHost}/colormaps/${gibsLayer.product}`)
-          const colorMapResponse = await response.json()
-
-          setColorMap({
-            ...colorMap,
-            [focusedCollectionId]: {
-              ...colorMap[focusedCollectionId],
-              [projection]: colorMapResponse
-            }
-          })
-        } catch (error) {
-          setColorMap({
-            ...colorMap,
-            [focusedCollectionId]: {
-              ...colorMap[focusedCollectionId],
-              [projection]: null
-            }
-          })
-        }
-      }
-    }
+  let colorMapState = {}
+  if (gibsTag && hasGibsLayerForProjection(gibsTag, projection)) {
```

Comment:
Would it make sense to pull `hasGibsLayerForProjection` into a util so we can more easily unit test?

Reply:
Added it to static/src/js/util/map/projections.js